tests/framework/Optimizers/GA/test_GA.xml

```diff
@@ -0,0 +1,207 @@
+          </mutation>
+        </reproduction>
+        <fitness type="invLinear">
+          <a>2.0</a>
+          <b>1.0</b>
+        </fitness>
+        <survivorSelection>fitnessBased</survivorSelection>
+      </GAparams>
+
+      <convergence>
+        <objective>20.0</objective>
+      </convergence>
+
+      <variable name="x1">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>1,2,3,4,5,6,7,8,9,10</initial>
+      </variable>
+
+      <variable name="x2">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>2,3,4,5,6,7,8,9,10,11</initial>
+      </variable>
+
+      <variable name="x3">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>3,4,5,6,7,8,9,10,11,12</initial>
+      </variable>
+
+      <variable name="x4">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>4,5,6,7,8,9,10,11,12,13</initial>
+      </variable>
+
+      <variable name="x5">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>5,6,7,8,9,10,11,12,13,14</initial>
+      </variable>
+
+      <variable name="x6">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>6,7,8,9,10,11,12,13,14,15</initial>
+      </variable>
+<!--
+      <variable name=" fuelLocation_3">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>3</initial>
+      </variable>
+
+      <variable name=" fuelLocation_4">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>4</initial>
+      </variable>
+
+      <variable name=" fuelLocation_5">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>5</initial>
+      </variable>
+
+      <variable name=" fuelLocation_6">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>6</initial>
+      </variable>
+
+      <variable name=" fuelLocation_7">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>7</initial>
+      </variable>
+
+      <variable name=" fuelLocation_8">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>8</initial>
+      </variable>
+
+      <variable name=" fuelLocation_9">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>9</initial>
+      </variable>
+
+      <variable name=" fuelLocation_10">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>10</initial>
+      </variable>
+
+      <variable name=" fuelLocation_11">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>11</initial>
+      </variable>
+
+      <variable name=" fuelLocation_12">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>12</initial>
+      </variable>
+
+      <variable name=" fuelLocation_13">
+        <distribution>uniform_dist_woRepl_1</distribution>
+        <initial>13</initial>
+      </variable> -->
```

Comment:
Remove?

Reply:
Um, it still seems to be a large block of commented out xml.